### ISSUE TITLE
Compliance scan shows warning for CBC ciphers enabled

### ISSUE THREAD

**Issue by scarolan** [chef-cft/bjc#477]
the ssl-baseline inspec profile was updated, and apparently our apache isn't quite secure enough to pass all its tests now.  Two things should happen:

1.  Kitchen updated to use the remote test instead of the locally stored one
2.  default-ssl.conf.erb updated to use the following cipher settings:

```
                SSLProtocol -all +TLSv1.2
                SSLCipherSuite EECDH+AESGCM:EDH+AESGCM
```

**Comment by ChefRycar** [comment]
Have tested this successfully. Have some other updates slated for today, and will see if I can't work this in.

**Comment by ChefRycar** [comment]
Update: Have determined that (1) is potentially unsafe. Ran into a repeatable error case where re-running verify on a re-converged node will get some interesting errors. Going to report in the repo, and leave the local testing in place for now. 

Note that this should *not* affect the nodes' compliance scans. Only the kitchen process seems to be hit by this edge case.

**Comment by ChefRycar** [comment]
Fix is included in 2.1.2. Leaving this ticket open until I can update the dev-sec repo with the bug I described above.

**Comment by scarolan** [comment]
pretty sure this is fixed now.
